rotkehlchen/accounting/accountant.py

```diff
@@ -280,6 +283,10 @@ def _process_event(
             )
             return 1, prev_time
 
+        if eth2_taxable_after_withdrawal_enabled is True and A_ETH2 in event_assets:
```

Comment:
nitpick: I would swap conditions since `A_ETH2 in event_assets` is shorter and thus the condition may become more readable